utils.py

```diff
@@ -73,10 +75,34 @@ def gen_json(metadata_dict, metadata_name):
 
   filename = metadata_name + '.json'
   fileobj = open(filename, 'w')
-  fileobj.write(json.encode_pretty_printed_json(metadata_dict))
+  fileobj.write(canonicaljson.encode_pretty_printed_json(metadata_dict))
   fileobj.close()
 
 
+def json_to_dict(json_filepath):
```

Comment:
Yep, good catch. Didn't know they already had that in there. Thanks Wilson, I'll switch to using the tuf function.